### ISSUE TITLE
citing from Da|RA database from inside editor of FW

### ISSUE THREAD

**Issue by afshinsadeghi** [OSCOSS/fiduswriter#68]
Similar to #66

**Comment by NiloofarAzizi** [comment]
to cite from Da|Ra database, datacite was used regarding filter option to Dara.

**Comment by afshinsadeghi** [comment]
@NiloofarAzizi 
can you point to the commit number and the URL you committed, please?

**Comment by afshinsadeghi** [comment]
this is a duplicate of #67 ,  I closed that one.

**Comment by afshinsadeghi** [comment]
Note from Philipp: searching and citing of datasets from DARA http://www.da-ra.de/home/

**Comment by afshinsadeghi** [comment]
@johanneswilm  can you confirm testing it? I want to close it if it is merged

**Comment by johanneswilm** [comment]
@sadeghiafshin I have not seen the code, but we worked closely on  questions relation to the implementation. In short: the dara-site is a bit of a mess and doesn't allow for serious work. But the dara data is exported to datacite, which has a consistent API and it is possible to filter for the data that only comes from Dara. So I advised Niloofar to use the datacite API and add the filter to only use dara data.  

Judging by this, it looks like noone has committed anything to any branch of fiduswriter for the past 18 days: https://github.com/fiduswriter/fiduswriter/network
